### PR TITLE
Parity with 2.0.0-beta.4

### PR DESCRIPTION
**Pull request by smalyala**

### CODE REVIEW

File: src/python_pachyderm/mixin/pfs.py [hunk 1]
```diff
@@ -605,12 +628,16 @@ def get_file(self, commit, path, URL=None):
...
         res = self._req(
             Service.PFS,
             "GetFileTAR",
             req=pfs_proto.GetFileRequest(
-                file=pfs_proto.File(commit=commit_from(commit), path=path), URL=URL
+                file=pfs_proto.File(commit=commit_from(commit), path=path, tag=tag),
+                URL=URL,
             ),
         )
         return PFSFile(FileTarstream(res))
```
Comment (albscui) [inline src/python_pachyderm/mixin/pfs.py:643]: As you mentioned, maybe we can just use `BytesIO` instead of `FileTarstream`.

**Review by albscui — APPROVED** [review]
LGTM. The comments are just nits.

File: src/python_pachyderm/mixin/pfs.py [hunk 2]
```diff
@@ -595,7 +596,29 @@ def copy_file(
...
 
-    def get_file(self, commit, path, URL=None):
+    def get_file(self, commit, path, tag=None, URL=None):
+        """
+        Returns a `BytesIO` object, containing the contents of a file stored
+        in PFS.
+
+        Params:
+
+        * `commit`: A tuple, string, or `Commit` object representing the
```
[the remark this answers is not in the document]
Comment (smalyala) [reply src/python_pachyderm/mixin/pfs.py:606]: Hmm, that's a good suggestion. Only concern is path a necessary component for commit?

File: src/python_pachyderm/mixin/pfs.py (hunk 1 above)
Comment (smalyala) [reply src/python_pachyderm/mixin/pfs.py]: +1

File: src/python_pachyderm/mixin/pfs.py [hunk 3]
```diff
@@ -595,7 +596,29 @@ def copy_file(
...
+        * `URL`: A string that specifies an object storage URL that the file
+        will be uploaded to.
+        """
+        res = self._req(
+            Service.PFS,
+            "GetFile",
+            file=pfs_proto.File(commit=commit_from(commit), path=path, tag=tag),
+            URL=URL,
+        )
+        return io.BytesIO(next(res).value)
```
[the remark this answers is not in the document]
Comment (smalyala) [reply src/python_pachyderm/mixin/pfs.py]: +1

**Pull request merged**